### PR TITLE
SetGridField sort fixes

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/39685

## Description
- Fixed an issue where the *sort_by* argument was not respected.
- Added support for multiple columns sort using the *sort_by* argument.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
@yaakovi 
`Packs/CommonScripts/Scripts/SetGridField/SetGridField.yml: [CJ105] - All test playbooks for SetGridField in this pack are currently skipped. Please unskip at least one of the relevant test playbooks.`
but It has no TPBs. familiar with it?

File: Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml [hunk 1]
```diff
@@ -108,5 +108,5 @@ script:
   type: python
   subtype: python3
 tests:
-- No tests - added unitest
+- No tests
```
Comment (yuvalbenshalom) [inline Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml:111]: why did you touch this?

Comment (bakatzir) [reply Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml:111]: to try and handle `CJ105` err. but maybe it is just a dev bug. WDYT?

Comment (bakatzir) [reply Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml:111]: sorry, wrong file my bad.

**Review by yuvalbenshalom — APPROVED** [review]

**Review by yaakovi — APPROVED** [review]
wait for the Gitlab issue to be resolve before resolving conflicts (by that time you ould probably get another conflict)

**Pull request merged**